packages/zoltan2/core/src/algorithms/color/Zoltan2_AlgHybridD1.hpp

```diff
@@ -0,0 +1,985 @@
+#ifndef _ZOLTAN2_ALGHYBRIDGMB_HPP_
+#define _ZOLTAN2_ALGHYBRIDGMB_HPP_
+
+#include <vector>
+#include <unordered_map>
+#include <iostream>
+#include <fstream>
+#include <queue>
+#include <sys/time.h>
+#include <algorithm>
+
+#include "Zoltan2_Algorithm.hpp"
+#include "Zoltan2_GraphModel.hpp"
+#include "Zoltan2_ColoringSolution.hpp"
+#include "Zoltan2_Util.hpp"
+#include "Zoltan2_TPLTraits.hpp"
+#include "Zoltan2_AlltoAll.hpp"
+
+#include "Tpetra_Core.hpp"
+#include "Teuchos_RCP.hpp"
+#include "Tpetra_Import.hpp"
+#include "Tpetra_FEMultiVector.hpp"
+
+#include "KokkosKernels_Handle.hpp"
+#include "KokkosKernels_IOUtils.hpp"
+#include "KokkosGraph_Distance1Color.hpp"
+#include "KokkosGraph_Distance1ColorHandle.hpp"
+#include <stdlib.h>
+
+//////////////////////////////////////////////
+//! \file Zoltan2_AlgHybridGMB.hpp
+//! \brief A hybrid version of the framework proposed by Gebremedhin and Manne
```

Comment:
Better to say "hybrid MPI+Kokkos" for clarity since hybrid could mean many things.